rkt/containers.go

```diff
@@ -0,0 +1,222 @@
+// Copyright 2014 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sort"
+	"syscall"
+	"time"
+
+	"github.com/coreos/rocket/pkg/lock"
+)
+
+type container struct {
+	*lock.DirLock
+	uuid       string
+	isExited   bool
+	isGarbage  bool
+	isDeleting bool
+	createdAt  time.Time
+	garbageAt  time.Time
+}
+
+type includeMask byte
+
+const (
+	includeContainersDir includeMask = 1 << iota
+	includeGarbageDir
+)
+
+var (
+	skipUUID = errors.New("skip this container")
```

Comment:
man I've already become spoiled by the tooling informing me about unused entities!